src/main/java/com/amazon/opendistroforelasticsearch/knn/index/v1736/KNNIndex.java

```diff
@@ -87,12 +87,6 @@ public long getIndexSize() {
     public void close() {
         Lock writeLock = readWriteLock.writeLock();
         writeLock.lock();
-
```

Comment:
Why is this no longer needed?

Reply:
Good catch. Bad merge. Will add it back.